client/js/components/home.jsx

```diff
@@ -87,7 +87,13 @@ class Home extends React.Component {
       this.resetHierarchy(bank, map);
     }
     let itemChecked = Object.assign({}, this.state.itemChecked, map);
-    this.setState({itemChecked});
+    this.setState({
```

Comment:
Would you please extract the setting of this state into a new function, and have the other code that sets isOpen to false call it as well (line 281)?